conf/reference/features_reference.conf

```diff
@@ -28,6 +28,9 @@ features {
   # Whether the tumblr intake process is supported
   intakeSupported = true
 
+  # Wether to enable redirect to intake checkbox by default
```

Comment:
A couple of minor changes:
 Whether to set the Redirect to Intake checkbox by default